app/views/layouts/application.html.haml

```diff
@@ -7,7 +7,7 @@
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
     = javascript_include_tag 'application', 'data-turbolinks-track' => true
     = csrf_meta_tags
-%body{ng:{app: "customerApp"}}
+%body{ ng: { controller: 'CheckoutController' }}
```

Comment:
@fiestacasey I wasn't sure how else to get order.order_items to be consistent and accessible throughout the entire app. Would an AppController that just contains the order be a better approach?